diff --git a/data/backend/boltz.conf b/data/backend/boltz.conf
index a07ff9d..7b9ab57 100755
--- a/data/backend/boltz.conf
+++ b/data/backend/boltz.conf
@@ -27,7 +27,7 @@ password = "boltz"
   port = 9005
 
 [swap]
-deferredClaimSymbols = ["BTC", "L-BTC", "RBTC"]
+deferredClaimSymbols = ["BTC", "L-BTC"]
 
 [[pairs]]
 base = "BTC"
@@ -39,11 +39,11 @@ maxSwapAmount = 40_294_967
 minSwapAmount = 50_000
 
   [pairs.timeoutDelta]
-  chain = 1440
-  reverse = 1440
-  swapMinimal = 1440
-  swapMaximal = 2880
-  swapTaproot = 10080
+  chain = 180
+  reverse = 180
+  swapMinimal = 400
+  swapMaximal = 450
+  swapTaproot = 1200
 
 [[pairs]]
 base = "L-BTC"
@@ -62,53 +62,11 @@ minSwapAmount = 100
   minSwapAmount = 25_000
 
   [pairs.timeoutDelta]
-  chain = 1440
-  reverse = 1440
-  swapMinimal = 1440
-  swapMaximal = 2880
-  swapTaproot = 10080
-
-[[pairs]]
-base = "RBTC"
-quote = "BTC"
-rate = 1
-fee = 0.25
-swapInFee = 0.1
-
-maxSwapAmount = 4_294_967
-minSwapAmount = 50_000
-
-  [pairs.timeoutDelta]
-  chain= 1440
-  reverse = 1440
-  swapMinimal = 1440
-  swapMaximal = 2880
-  swapTaproot = 10080
-
-[[pairs]]
-base = "RBTC"
-quote = "L-BTC"
-rate = 1
-fee = 0.25
-swapInFee = 0.1
-
-maxSwapAmount = 10_000_000
-minSwapAmount = 2_500
-
-swapTypes = ["chain"]
-
-  [pairs.chainSwap]
-  buyFee = 0.1
-  sellFee = 0.1
-
-  minSwapAmount = 25_000
-
-  [pairs.timeoutDelta]
-  chain= 1440
-  reverse = 1440
-  swapMinimal = 1440
-  swapMaximal = 2880
-  swapTaproot = 10080
+  chain = 180
+  reverse = 180
+  swapMinimal = 400
+  swapMaximal = 450
+  swapTaproot = 1200
 
 [[currencies]]
 symbol = "BTC"
@@ -171,19 +129,3 @@ maxZeroConfAmount = 40_294_967
   feeFloor = 0.1
 
   wallet = "regtest"
-
-[rsk]
-networkName = "Anvil"
-providerEndpoint = "ws://anvil:8545"
-
- [[rsk.contracts]]
- etherSwap = "0x8464135c8F25Da09e49BC8782676a84730C318bC"
- erc20Swap = "0x71C95911E9a5D330f4D621842EC243EE1343292e"
-
- [[rsk.tokens]]
- symbol = "RBTC"
-
- maxSwapAmount = 4_294_96700
- minSwapAmount = 10000
-
- minWalletBalance = 100_000_000
